resmo/README.md

```diff
@@ -0,0 +1,59 @@
+## Overview
+[Resmo][1] is a continuous asset visibility, security, and compliance solution for cloud and SaaS. Integrate with your cloud and SaaS tools to start querying and securing your resources. Monitor them in one place, run security and compliance audits, and get alerted on violations.
+
+What does Resmo offer to Datadog users?
+- Collect and monitor all your Datadog assets on a single platform.
+- Query your Datadog monitors, roles, permissions, API Keys, and more.
+- Set up rules to continuously assess your Datadog resources.
+- Set up and receive rule notifications based on your Datadog resource configurations.
+
+
+## Setup
+
+1. Go to https://resmo.com, log in and navigate to **Integrations**.<br/>
+
+![Resmo Screenshot][2]
+
+2. Click **Add Integration** and select **Datadog**. Write a descriptive name, and optionally a description.<br />
+
+![Resmo Screenshot][3]
+
+3. Go to your Datadog account and create a new organization [API key][12] and a user [Application key][13].<br />
+
+4. Return to your Resmo account and enter the copied keys into the Organization API Key and User App Key fields. <br />
+
+![Resmo Screenshot][7]
+
+5. Select a site from the dropdown menu on the last field.
+
+![Resmo Screenshot][8]
+
+6. Click the **Create** button, and you're ready to query your Datadog resources.
```

Comment:
This section will need to be updated to instruct the user to instead navigate to the Resmo tile in Datadog, and click Connect Accounts to kick off OAuth.